service/README.rst

```diff
@@ -104,12 +104,11 @@ access to any subset of the privileged PlatformIO features.
 Status
 ------
 
-The GEOPM systemd service is a new feature.  What exists currently is
-a work in progress, and there are several outstanding issues that must
-be resolved before the GEOPM Service is ready for release with GEOPM
-version 2.0.  These
+The GEOPM systemd service is a new feature for version 2.0.  The
+``v2.0.0-rc1`` tag is the first release candidate and for 2.0 and
+provides the GEOPM Service feature.  Tagging ``v2.0.0`` will not occur
+until all issues with the "2.0" label are closed or relabeled:
```

Comment:
Minor suggestion:

For some reason, the wording makes it feel like this is providing process-level details. If the intention is to give an idea of when v2.0.0 will be tagged and/or the features/changes that will be included in v2.0.0, maybe it could be worded something like:
- "The v2.0.0 tag/release will include the following changes: <link to all 2.0 issues>. For a list of remaining issues/changes, please see: <link to remaining 2.0 issues>"
- "Work for v2.0.0 is currently ongoing and will be tagged once all remaining issues/changes are completed/addressed: <link to remaining 2.0 issues>"